lua/wikis/commons/Info.lua

```diff
@@ -29,6 +29,7 @@ return {
 			hasPosition = false,
 			hasSpecialTeam = false,
 			allowManual = true,
+			defaultPlayerNumber = 5,
```

Comment:
why are we pushing that setting into the squads config?
isn't it unrelated to squad tables?